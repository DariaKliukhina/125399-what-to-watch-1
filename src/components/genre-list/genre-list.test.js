import React from 'react';
import renderer from 'react-test-renderer';
import GenreList from './genre-list.jsx';


it(`GenreList screen renders correctly`, () => {
  const mocks = {
    genre: `All genres`,
    functionHandler: jest.fn()
  };

  const component = renderer.create(
      <GenreList
        activeGenre={mocks.genre}
        onGenreClick={mocks.functionHandler}
      />
  ).toJSON();

  expect(component).toMatchSnapshot();
});

