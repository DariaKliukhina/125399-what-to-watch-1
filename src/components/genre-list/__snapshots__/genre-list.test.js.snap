// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreList screen renders correctly 1`] = `
<li
  className="catalog__genres-item"
>
  <a
    className="catalog__genres-link"
    href="#"
  >
    Comedies
  </a>
</li>
`;
